src/main/java/com/bina/varsim/fastqLiftover/MafRecord.java

```diff
@@ -0,0 +1,51 @@
+package com.bina.varsim.fastqLiftover;
+
+import java.util.ArrayList;
+
+public class MafRecord {
+    public class MafEntry {
+        //https://cgwb.nci.nih.gov/FAQ/FAQformat.html#format5
+        public final String src;
+        public final int start0;
```

Comment:
I guess this is zero-based? please add a comment. We should eventually move to Austin's suggestion of building types for each kind of value